lib/beaker/dsl/install_utils.rb

```diff
@@ -1453,6 +1453,57 @@ def build_ignore_list(opts = {})
         ignore_list << '..' unless ignore_list.include? '..'
         ignore_list
       end
+
+      # This method is a fix for PUP-2365 when a windows agent cannot download
+      # modules from the forge because it doesn't have the appropriate
+      # certificates installed to validate the https connection.
+      #
+      # This method installs the certifcates required in order to that functionality
+      # to work as expected.
+      #
+      # @param [Host] host             A host object
+      #
+      def install_forge_cert(host)
+        geotrust_global_ca = <<-CERT.gsub(/^ {8}/, '')
+        -----BEGIN CERTIFICATE-----
+        MIIDVDCCAjygAwIBAgIDAjRWMA0GCSqGSIb3DQEBBQUAMEIxCzAJBgNVBAYTAlVT
```

Comment:
I really don't believe that this cert belongs in the beaker codebase.  I'm fine with having a helper function for installing certs, but you should have the cert in questions as part of a pre-suite set up that you do.